src/server/index.ts

```diff
@@ -172,10 +171,6 @@ const startLanguageServer = (
     return handleHover(document, params);
   });
 
-  if (reader) {
-    setFileSystemReader(reader);
```

Comment:
I might be missing something, but where in the code are we actually passing the new argument?

Reply:
Oops... thanks for catching this!